en-US/Punctuation.xml

```diff
@@ -647,9 +767,21 @@ PLAY [Gather and display facts for managed nodes] ****************************</
 		  <para>
             The path contains the library qualifier in braces.
           </para>
+          </example>
+		<example><title>Referring to Special Characters</title>
+
+          <para>
+            When a regular user starts a shell, the prompt includes an ending dollar character ($).
```

Comment:
I would use backticks on the dollar sign in the parentheses, since it's part of a command prompt.